crates/ty/tests/cli/snapshots/cli__output_format_gitlab.snap

```diff
@@ -0,0 +1,55 @@
+---
+source: crates/ty/tests/cli/main.rs
+info:
+  program: ty
+  args:
+    - check
+    - "--output-format"
+    - gitlab
+    - "--target-version"
+    - "3.9"
+---
+success: false
+exit_code: 1
+----- stdout -----
+[
+  {
+    "check_name": "revealed-type",
+    "description": "revealed-type: Revealed type: `LiteralString`",
+    "severity": "info",
+    "fingerprint": "979dad1e810c03b2",
```

Comment:
You have to redact the `fingerprint`, I think, because it might be different on different platforms? At least, that's what you did in https://github.com/astral-sh/ruff/blob/ed64c4d943d5a29c017550a6fc015922666002ac/crates/ty/tests/cli/main.rs#L656